binaries/conda/build_packages.py

```diff
@@ -6,12 +6,13 @@
 conda_build_dir = os.path.dirname(os.path.abspath(__file__))
 REPO_ROOT = os.path.join(conda_build_dir, "..", "..")
 MINICONDA_DOWNLOAD_URL = "https://repo.anaconda.com/miniconda/Miniconda3-py39_4.9.2-Linux-x86_64.sh"
+CONDA_BINARY = f"$HOME/miniconda/condabin/conda"
```

Comment:
miniconda is not necessary as long as conda is installed. Here, it is better to check conda is available first.

Reply:
In a codebuild job, where this code will primarily run, miniconda/anaconda is not available by default. Hence, it needs to be installed. In the rare-case scenario where this script maybe run on an already-existing conda environment, installing miniconda will help because the miniconda version is pinned here, which makes the build process more predictable. I believe this should be retained.

Reply:
In EC2, conda is already installed. The pseudocode is sth like. 
if conda exists
    CONDA_BINARY = conda path
else
    install miniconda
    CONDA_BINARY = minicode path